src/control/server/storage/faults.go

```diff
@@ -265,6 +266,15 @@ func FaultPathAccessDenied(path string) *fault.Fault {
 	)
 }
 
+// FaultInvalidSPDKConfig creates a Fault for the case where SPDK configuration is invalid.
+func FaultInvalidSPDKConfig(err error) *fault.Fault {
+	return storageFault(
+		code.SpdkInvalidConfiguration,
+		fmt.Sprintf("unable to parse SPDK configuration: %s", err),
+		"regenerate the configuration and restart daos_server",
```

Comment:
suggestion - add some detail on how to do that or where in the admin guide to find this information

Reply:
I don't expect this to be surfaced to the admin unless something is really screwed up. It has to be a fault so that it transits the daos_server <-> daos_server_helper boundary cleanly. I initially started with a simple sentinel error, but in that case the server would only see a stringified representation of the error.

In any case, I'm not sure what we could document here... There isn't a manual procedure for regenerating the config aside from storage format, so IMO if this fault somehow surfaces, it means something has gone wrong in the code and it's a support engagement. I'd prefer to keep the resolution string as-is because it's generic enough to not need updating later but specific enough to direct support toward a potential resolution.

Reply:
Okay, that makes sense. I think my brain saw that it was a Fault and assumed it must become user visible at some point. Considering we always catch it, the vagueness is fine.